src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import _ from 'lodash';
+
+import { veteranBenefitSummaryOptionText } from '../utils/helpers';
+import { updateBenefitSummaryOption } from '../actions/letters';
+
+class VeteranBenefitSummaryLetter extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(domEvent) {
+    this.props.updateBenefitSummaryOption(domEvent.target.id, domEvent.target.checked);
+  }
+
+  render() {
+    const benefitInfo = this.props.benefitSummaryOptions.benefitInfo;
+    const optionsToInclude = this.props.optionsToInclude;
+    let militaryServiceInfo;
+    let vaBenefitInformation;
+    let vaBenefitInfoRows = [];
+
+    _.forIn(benefitInfo, (value, key) => {
+      const optionText = veteranBenefitSummaryOptionText(benefitInfo, key);
+
+      // There are 2 conditions this is checking for
+      // 1. If the current benefit is not in the option text list from helpers,
+      // it means this benefit is not for veterans, and therefore it shouldn't
+      // be displayed.
+      // 2. If the value of the current benefit is false, we don't want to
+      // display it. Values can be either true, false, or some other value,
+      // which is why we're checking that it's not false.
+      if (optionText && value !== false) {
+        vaBenefitInfoRows.push(
+          <tr key={`option${key}`}>
+            <th scope="row">
+              <input
+                  aria-labelledby={`${key}Label`}
+                  autoComplete="false"
+                  checked={optionsToInclude[key]}
+                  id={key}
+                  name={key}
+                  type="checkbox"
+                  onChange={this.handleChange}/>
+            </th>
+            <td><label id={`${key}Label`}>{veteranBenefitSummaryOptionText(benefitInfo, key)}</label></td>
+          </tr>
+        );
+      }
+    });
+
+    if (this.props.optionsAvailable) {
+      vaBenefitInformation = (
+        <table className="usa-table-borderless">
+          <thead>
+            <tr>
+              <th scope="col">Include</th>
+              <th scope="col">Something</th>
+            </tr>
+          </thead>
+          <tbody>
+            {vaBenefitInfoRows}
+          </tbody>
+        </table>
+      );
+    } else {
+      vaBenefitInformation = (
+        <div className="feature">
+          <h4>VA Benefit Information is currently unavailable</h4>
+          <div>Please try again later. You can call the Vets.gov Help Desk for additional assistance.</div>
+          <div>1-855-574-7286</div>
+          <div>Monday - Friday, 8:00am - 8:00pm (ET)</div>
```

Comment:
@phiden could you provide feedback on this language? This message appears if the VA benefit information options aren't available for some reason.